pvlib/modelchain.py

```diff
@@ -1068,6 +1076,51 @@ def faiman_temp(self):
     def fuentes_temp(self):
         return self._set_celltemp(self.system.fuentes_celltemp)
 
+    @property
+    def dc_ohmic_model(self):
+        return self._dc_ohmic_model
```

Comment:
any idea why codecov says this line isn't tested? It looks like it should be.

Reply:
That was confusing for me as well. And now it is telling me that blank lines 1082 and 1095 are not covered, and that 1090 is not covered which it should be as there is a test specifically for `dc_ohmic_model='no_loss'`. I'd welcome an assist from someone more familiar with the tests on what might be leading to the failed coverage if anyone has an idea.

Reply:
I ran a coverage report locally and cannot reproduce the codecov results. Adding exceptions to these lines causes errors, as you would expect.